explorer/src/components/Footer.tsx

```diff
@@ -136,6 +155,14 @@ const Footer = () => (
               >
                 Blog
               </Link>
+              <Link
+                href={brand}
```

Comment:
This is an internal link so should use `component={RouterLink}` and `to` like the above.